rpc/rpc.go

```diff
@@ -157,6 +157,12 @@ func (l *Link) RPC(ctx context.Context, msg *amqp.Message) (*Response, error) {
 	}
 	msg.Properties.ReplyTo = l.clientAddress
 
+	if msg.ApplicationProperties["server-timeout"] == nil {
```

Comment:
[nit] In the rest of the code base, we use the `_, ok` syntax to check for missing keys.

``` Go
if _, ok := msg.ApplicationProperties["server-timeout"]; !ok {
    // More code here.
}
```

Reply:
Changed the syntax for missing key check.

Reply:
Also. should I change [line](https://github.com/Azure/azure-amqp-common-go/blob/f224da4b2977f9b634f33cafc49c2cf57f1eaaa9/rpc/rpc.go#L155) to comply with the codebase syntax?

Reply:
Message is a struct, not a map. Therefor the `_, ok` syntax is inapplicable to that line.